src/main/java/seedu/duke/commands/member/TeamMemberDeleteCommand.java

```diff
@@ -0,0 +1,37 @@
+package seedu.duke.commands.member;
+
+import seedu.duke.DukeExceptions;
+import seedu.duke.commands.Command;
+import seedu.duke.member.TeamMember;
+import seedu.duke.project.Project;
+import seedu.duke.ui.Ui;
+
+import java.util.ArrayList;
+
+public class TeamMemberDeleteCommand extends Command {
+    private int memberIndex;
+
+    public TeamMemberDeleteCommand(int memberIndex) {
+        this.memberIndex = memberIndex;
+    }
+
+    @Override
+    public String executeCommand(ArrayList<Project> projects,
```

Comment:
It becomes important for members to keep track of tasks so deleted members will no longer be attached to tasks.

Reply:
I think as of now that feature is not in our user guide. Like showing the list of tasks attached to members.
So since no other command uses it for now I didn't implement it.
It's a breadth first approach